packages/backend/src/studio-api-impl.ts

```diff
@@ -359,8 +369,13 @@ export class StudioApiImpl implements StudioAPI {
     return this.taskRegistry.getTasks();
   }
 
-  async openVSCode(directory: string): Promise<void> {
+  async openVSCode(directory: string, recipeId?: string): Promise<void> {
+    const telemetry = {
+      'recipe.id': recipeId,
+    };
     if (!path.isAbsolute(directory)) {
+      telemetry['errorMessage'] = 'Do not support relative directory.';
+      this.telemetry.logUsage('studio.open-vscode', telemetry);
```

Comment:
Would have been better to have a wider try catch finally and have a single call to telemetry

Reply:
Done